### PR TITLE
setup,usage: Port from ciao-cli to ciao

### PR DESCRIPTION
**Pull request by rbradford**
Update the documentation to cover the port from ciao-cli to ciao.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Comment by rbradford** [comment]
@markdryan I left scripting.md for you as you're the expert there!

**Comment by rbradford** [comment]
CC @kaccardi

File: ciao/usage/ciao.md [hunk 1]
```diff
@@ -6,89 +6,34 @@ keywords: CIAO, command line, tool
...
+`ciao` requires Cloud Integrated Advanced Orchestrator specific
 environment variables to retrieve credentials and networking information:
 
-* `CIAO_CONTROLLER` exports the controller URL
+* `CIAO_CONTROLLER` provides the controller URL
 * `CIAO_CLIENT_CERT_FILE` provides the certificate to authenticate against the controller
 * `CIAO_CA_CERT_FILE` (optional) use the supplied certificate as the CA
 
-All those environment variables can be set through an rc file.
+All those environment variables can be set through an sourced file.
```
Comment (markdryan) [inline ciao/usage/ciao.md]: a sourced file

File: ciao/usage/tutorial.md [hunk 2]
```diff
@@ -258,14 +238,15 @@ instances when creating them our Guestbook container would not be able to
...
-ciao-cli instance list command.
+`ciao list instances` command.
 
 ```shell
-$ ciao-cli instance list
-# UUID                                 Status Private IP SSH IP        SSH PORT
-1 2111f605-d02c-489d-acf0-05a10d24833b active 172.16.0.2 198.51.100.84 33002
-2 6bcf6caa-a77e-4fe2-b227-c9bd4122340a active 172.16.0.3 198.51.100.84 33003
-3 b40f3aab-908f-4377-a6bf-e9b7703a0684 active 172.16.0.4 198.51.100.84 33004
+rob@singlevm:~$ ciao list instances
```
Comment (markdryan) [inline ciao/usage/tutorial.md]: Elsewhere we just used $ as the prompt.

**Review by markdryan — COMMENTED** [review]
LGTM.  Just two minor comments.

**Comment by rbradford** [comment]
Thanks @markdryan. Done!

**Pull request merged**